src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -539,9 +539,10 @@ private void QueryResults()
                 _updateToken = currentCancellationToken;
                 var queryText = QueryText.Trim();
 
-                var pluginQueryPairs = QueryBuilder.Build(ref queryText);
+                var pluginQueryPairs = QueryBuilder.Build(queryText);
                 if (pluginQueryPairs != null && pluginQueryPairs.Count > 0)
                 {
+                    queryText = pluginQueryPairs.Values.First().ActionKeyword;
```

Comment:
This used to be the whole query minus the action keyword, now it's just the keyword?

Reply:
Good catch!

Reply:
It was a normalized query with an action keyword